### PR TITLE
document JVM option MaxFDLimit for macOS

### PR DESCRIPTION
**Pull request by stuvie**
This Pull request fixes issue #26899

### CODE REVIEW

File: docs/reference/setup/sysconfig/file-descriptors.asciidoc [hunk 1]
```diff
@@ -16,6 +16,9 @@ For the `.zip` and `.tar.gz` packages, set <<ulimit,`ulimit -n 65536`>> as
 root before starting Elasticsearch,   or set `nofile` to `65536` in
 <<limits.conf,`/etc/security/limits.conf`>>.
 
+On macOS, you must also pass the JVM option <<-XX:-MaxFDLimit>>
```
Comment (jasontedor) [inline docs/reference/setup/sysconfig/file-descriptors.asciidoc]: The `<<>>` syntax is for referring to other sections in the docs. Here we can use backticks ` surrounding `-XX:-MaxFDLimit` to get the formatting that I think you're looking for.

**Review by jasontedor — COMMENTED** [review]
Thanks for the PR, I left one comment. Can you make the change?

**Comment by stuvie** [comment]
Thanks @jasontedor - fixed (and thanks for your time at the conference last week)

**Review by jasontedor — APPROVED** [review]
LGTM.

**Comment by jasontedor** [comment]
Thanks @stuvie, I've integrated this change.

> (and thanks for your time at the conference last week)

It was my pleasure, and thanks for attending and your interest in Elasticsearch.

**Pull request merged**